.gitignore

```diff
@@ -25,4 +25,6 @@ venv
 config.toml
 .envrc
 qml_debug.log
-datasets/*
\ No newline at end of file
+datasets/*
+coverage.xml
```

Comment:
Is this generated when benchmarking too?

Reply:
No. The first one was formatted to remove an extra space at the end (this is part of my VSCode configuration).
The second one should be there, and I included it because I was testing the coverage and noticed that it was not. I can remove the coverage file if you prefer.